packages/pigment-css-vite-plugin/src/index.ts

```diff
@@ -66,10 +66,10 @@ export function pigment(options: PigmentOptions) {
       name: 'pigment-css-theme-injection-plugin',
       enforce: 'pre',
       resolveId(source) {
-        if (finalTransformLibraries.some((lib) => source.includes(`${lib}/styles.css`))) {
+        if (allLibs.some((lib) => source.includes(`${lib}/styles.css`))) {
```

Comment:
Could you move this check to a util function, use that here and add some test that were previously failing.